bonsai_raw_data_schema.sql

```diff
@@ -37,10 +37,21 @@ CREATE TABLE "temporal_extent" (
   "finishes" date
 );
 
+
+CREATE TABLE "datasource" (
+  "id" SERIAL PRIMARY KEY,
+  "label" TEXT,
+  "version" TEXT NOT NULL,
+  "source" TEXT,
```

Comment:
Please add text here indicating how to use this field

Reply:
Can we add a note here that this is a draft proposal until https://github.com/BONSAMURAIS/schema/issues/9 is resolved with more detail and a formal process?